packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-overview/index.md

```diff
@@ -12,7 +12,7 @@ Using the sample app as your guide, walk through each of the use cases in this s
 The supported use cases are as follows:
 
 * [Load the widget](/docs/guides/oie-embedded-widget-use-cases/aspnet/oie-embedded-widget-use-case-load/)
-* [Basic sign-in](/docs/guides/oie-embedded-widget-use-cases/aspnet/oie-embedded-widget-use-case-basic-sign-in/)
+* [Basic sign in](/docs/guides/oie-embedded-widget-use-cases/aspnet/oie-embedded-widget-use-case-basic-sign-in/)
```

Comment:
sign-in is used as a noun here. I'd keep the hyphen.

Reply:
Title changed to: Basic sign-in flow using the Widget. I forgot to update this line. Thanks